app/controllers/businessTradingName/CheckYourAnswersController.scala

```diff
@@ -82,7 +82,7 @@ class CheckYourAnswersController @Inject() (val errorHandler: ErrorHandler,
             OK,
             successModel.formBundle
           ),
-          Some(pageUrl)
+          Some(controllers.routes.ChangeSuccessController.tradingName.url)
```

Comment:
With this change the `pageUrl` parameter in this function is now unused - can you remove that and then update the two places that call this func so that they don't pass the param